e2e_playwright/conftest.py

```diff
@@ -442,8 +442,7 @@ def fulfill_iframe_request(route: Route) -> None:
             # want to download a CSV via the blob: url; Chrome seems to be more lax
             frame_src_blob = ""
             if browser is not None and (
-                browser.browser_type.name == "webkit"
-                or browser.browser_type.name == "firefox"
+                browser.browser_type.name in {"webkit", "firefox"}
```

Comment:
nit: do you know why here the syntax is `... in {"...", "..."}` with curly braces but [here](https://github.com/streamlit/streamlit/pull/11216/files#diff-ffa3420ad895cbeab6d622382ad10e372ea092d2f2a1780ec3255b00c1ca893eR38) it is `level in ("CRITICAL", logging.CRITICAL)` with normal brackets?

Reply:
Not sure, this is one of the automatic fixes. Maybe it's only using set if it's the same file type, and tuple instead? I think both should probably work well

Reply:
I changed it to set it to make it a bit more unified.